app/build.gradle

```diff
@@ -92,9 +89,10 @@ dependencies {
 }
 
 buildscript {
-    ext.kotlin_version = '1.1.1'
+    ext.kotlin_version = '1.1.2-3'
     repositories {
         jcenter()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap-1.1' }	
```

Comment:
just curious, what does this repo provide that's not in the regular 1.1 version of kotlin?